src/components/Form/EditDiscountForm/EditDiscountForm.tsx

```diff
@@ -322,12 +349,17 @@ const EditDiscountForm = () => {
                     className="px-14 mr-4"
                     color="primary"
                     tag="button"
+                    disabled={
```

Comment:
Here it will become:

```tsx
<Button
  type="submit"
  className="px-14 mr-4"
  color="primary"
  tag="button"
  disabled={areObjectsEqual(
    { ...initialValues },
    { ...values }
   )}
>
  Salva
</Button>
```

**NB**: I'm passing a copy of the two elements otherwise the lodash functions mutate them as reference